client_app/src/components/shelter/RepeatableShifts.js

```diff
@@ -0,0 +1,207 @@
+// client_app/src/components/shelter/RepeatableShifts.js
+
+import React, { useState } from "react";
+import { useParams } from "react-router-dom";
+import "../../styles/shelter/RepeatableShifts.css";
+import { scheduleAPI } from "../../api/schedule";
+
+
+const RepeatableShifts = () => {
+  const { shelterId } = useParams(); // grab the shelterId from URL
+
+  const [shifts, setShifts] = useState([]);
+  const [newShift, setNewShift] = useState({
+    shiftName: "",
+    startTime: "",
+    endTime: "",
+    requiredVolunteers: 1,
+    maxVolunteers: 5,
```

Comment:
These field names are inconsistent with what's expected by the server. The server expects each ServiceShift to have the following fields (some have default values)
    shelter_id: str
    shift_start: int  
    shift_end: int
    shift_name: str = "Default Shift"
    required_volunteer_count: int = 1
    max_volunteer_count: int = 5